apps/mull-api/src/app/event/event.service.ts

```diff
@@ -76,23 +76,29 @@ export class EventService {
       .getMany();
   }
 
-  async create(eventInput: CreateEventInput): Promise<Event> {
-    return await this.eventRepository.save(eventInput);
+  async createEvent(input: CreateEventInput): Promise<Event> {
+    return await this.eventRepository.save(input);
+  }
+
+  async updateEvent(input: UpdateEventInput): Promise<Event> {
+    await this.eventRepository.update(input.id, { ...input });
+    return this.event(input.id);
   }
 
-  async update(eventInput: UpdateEventInput): Promise<Event> {
-    await this.eventRepository.update(eventInput.id, { ...eventInput });
-    return this.findOne(eventInput.id);
+  async deleteEvent(id: number): Promise<Event> {
+    const event = await this.event(id);
+    await this.eventRepository.delete(event.id);
+    return event;
   }
 
-  async addParticipant(eventId: number, userId: number) {
+  async joinEvent(eventId: number, userId: number) {
```

Comment:
I feel `eventService.addEventParticipant()` is more descriptive than `eventService.joinEvent()`
```suggestion
  async addEventParticipant(eventId: number, userId: number) {
```

Reply:
If we change this, we should change the below to `removeEventParticipant`

Reply:
That seems like a great name! I'll use your suggestion